content/php/concepts/math-functions/tan/tan.md

```diff
@@ -0,0 +1,30 @@
+#tan()
```

Comment:
You are missing the metadata for this entry. You can find a description of how the metadata is constructed in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata). Also yo need to remove the h1 header. The Title in the metadata handles the header in the entry.